### PR TITLE
Lonestar Oddities

### PR DESCRIPTION
**Pull request by CormanQuewye**
-Adds new oddities, unique to Lonestar, a few having gimmicks;
-The oddities can be acquired through the Luxury Imports Crate, under recreation;
-Most of these oddities are generally weak in power, and fairly expensive (the crate costs 1200 and brings 4), as to not tread too much on the Prospector's feet;
-They cannot get perks
-This should hopefully lead to more traffic in Lonestar, as it'll give more players direct access to (weak) oddities, and psions can gamble on potentially getting a good weapon out of it
-You opened the box
-A few more are in the works, but I can't be bothered to finish them this run.

### CODE REVIEW

File: code/game/objects/items/oddities.dm [hunk 1]
```diff
@@ -591,3 +591,259 @@
...
+	desc = "It has the sweetest aroma. You feel the flutter of butterflies."
+	icon_state = "bouquet"
+	item_state = "bouquet"
+	w_class = ITEM_SIZE_NORMAL
+	oddity_stats = list(
+		STAT_TGH = 6,
+	)
+
+/obj/item/oddity/ls/magazine
+	name = "Porn Magazine"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/oddities.dm:687]:
```suggestion
	name = "Swimming Magazine"
```

File: code/game/objects/items/oddities.dm [hunk 2]
```diff
@@ -591,3 +591,259 @@
...
+	desc = "A good, old porn magazine. Nowadays, it's mostly for collection purposes."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/oddities.dm:688]:
```suggestion
	desc = "A good, swimming magazine. Nowadays, it's mostly for collection purposes."
```

File: code/modules/cargo/packs_recreation.dm [hunk 3]
```diff
@@ -129,4 +129,14 @@
...
 	crate_name = "toy weapons crate"
+	group = "Recreation"
+
+/datum/supply_pack/oddities
+	name = "Luxury Imports Crate"
+	contains = list(/obj/random/lonestar_oddities,
+					/obj/random/lonestar_oddities,
+					/obj/random/lonestar_oddities,
+					/obj/random/lonestar_oddities)
+	cost = 1200
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs_recreation.dm:140]:
```suggestion
	cost = 1980
```

File: code/game/objects/items/oddities.dm [hunk 4]
```diff
@@ -591,3 +591,259 @@
...
+	icon_state = "detonator"
+	var/inert = 0
+	var/boom = 1
+	oddity_stats = list(
+		STAT_MEC = 6,
+		STAT_TGH = 3,
+	)
+
+/obj/item/oddity/ls/inertdetonator/attack_self(mob/user as mob)
+	if(inert == 1)
```
Comment (r4d6) [inline code/game/objects/items/oddities.dm:826]:
```suggestion
	if(inert)
```
Same thing, less stuff.

File: code/game/objects/items/oddities.dm [hunk 5]
```diff
@@ -591,3 +591,259 @@
...
+	else
+		on = TRUE
+		icon_state = "projector_on"
+		to_chat(user, "<span class='info'>The [src] shows you the beauty of the galaxy. A host of worlds, now beyond your reach.</span>")
+
+/obj/item/oddity/ls/inertdetonator //todo stop being lazy and give it a closed sprite
+	name = "Inert Detonator"
+	desc = "A device that was once in the hand of a terrorist; its payload is long dead. It's a miracle this thing ever worked, still, it's cool to look at!"
+	icon_state = "detonator"
+	var/inert = 0
```
Comment (r4d6) [inline code/game/objects/items/oddities.dm:818]:
```suggestion
	var/inert = FALSE
```

File: code/game/objects/items/oddities.dm [hunk 6]
```diff
@@ -591,3 +591,259 @@
...
+		return
+
+	if(prob (boom))
+		to_chat(user, SPAN_WARNING("The [src] clicks! Uh oh."))
+		explosion(src.loc, -1, 1, 4, 10)
+		qdel(src)
+
+	else
+		to_chat(user,("<span class='info'> The [src] clicks! Neat.</span>"))
+		inert = 1
```
Comment (r4d6) [inline code/game/objects/items/oddities.dm:837]:
```suggestion
		inert = TRUE
```

File: code/game/objects/items/oddities.dm [hunk 7]
```diff
@@ -591,3 +591,259 @@
...
+		on = FALSE
+	else
+		icon_state = "newtoncradle-on"
+		on = TRUE
+
+/obj/item/oddity/ls/puzzlebox
+	name = "Puzzlebox" //todo add cool sound
+	desc = "Looks oddly similar to a positronic brain. Something tells you you're better off not solving it."
+	icon_state = "puzzlebox"
+	var/openchance = 0.20
```
Comment (r4d6) [inline code/game/objects/items/oddities.dm:774]:
```suggestion
	var/openchance = 0.2
```

**Review by r4d6 — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs_recreation.dm (hunk 3 above)
Comment (CormanQuewye) [reply code/modules/cargo/packs_recreation.dm:140]: A bit much, I think. At 1200 a crate, that's 300 an oddity. These oddities are all pretty weak and even the ones that have a gimmick (like the Kriosan officer sword), don't really have a gimmick that skyrockets them to super useful. At 1980 (har, har), that's roughly 500 an oddity, so 600 for any noticeable return on sale. Seeing as this is meant to add traffic to Lonestar and further cement them as the bargain bin version of every other department, not sure increasing the cost is worth it. _Not to mention I have a handful of oddities left that aren't useful at all and will be further weighing the pool towards weakness._

File: code/game/objects/items/oddities.dm (hunk 2 above)
Comment (CormanQuewye) [reply code/game/objects/items/oddities.dm:688]: Kind of out of theme. If you want to remove the blatant adult reference, maybe something along the lines of:
	name = "Magazine"
	desc = "You bought this for the articles!"

**Comment by Kazkin** [comment]
There are lore conflicts with this PR. Oddities are unique to the planet and lose their power if taken outside the planet, though I don't think I wrote this down so that is an oversight on my end. 

However.

I like it, the idea, and the effort and think its fitting. We can handwave them being outside the planet by saying they only become oddities when they arrive or were held in orbit/within the planet's anomalous field.

**Pull request merged**